server/server.js

```diff
@@ -8,27 +8,23 @@ const cors = require("cors");
 
 const app = express();
 
-app.use(cors());
+app.use(cors({ origin: "*" }));
 
-app.set("view engine", "ejs");
+app.set("view engine", "pug");
 app.use(express.json());
```

Comment:
Disable autocomplete for sensitive fields. While this doesn't directly apply here, consider whether forms rendered by 'pug' contain sensitive information and disable autocomplete accordingly.

```suggestion
app.set("view engine", "pug");
```